jvm/buildSrc/src/main/java/com/radixdlt/cargo/desktop/DesktopTargetTriple.kt

```diff
@@ -33,10 +33,21 @@ sealed interface DesktopTargetTriple {
     }
 
     companion object {
-        val all = listOf(DarwinAArch64, DarwinX8664, LinuxArmel, LinuxX8664, LinuxWin32X8664)
+        private val all = listOf(
+            DarwinAArch64,
+            DarwinX8664,
+            LinuxArmel,
+            LinuxX8664,
+            LinuxWin32X8664
+        )
 
-        // Currently our CI supports building on Mac OS and we don't care about Mac Intel for now.
-        // So we skip the rest.
-        val supported = listOf(DarwinAArch64)
+        val ciSupported = listOf(
```

Comment:
If we ever need to build for other os, for example windows, we can add more to that and also make sure `release-desktop.yml` includes an entry to that host in the matrix.